modules/@angular/common/test/forms-deprecated/validators_spec.ts

```
@@ -143,14 +142,14 @@ export function main() {
         return (c: any /** TODO #9100 */) => {
           var emitter = new EventEmitter();
           var res = c.value != expected ? response : null;
-
-          PromiseWrapper.scheduleMicrotask(() => {
-            ObservableWrapper.callEmit(emitter, res);
+          Promise.resolve(null).then(() => {
+            emitter.emit(res);
             // this is required because of a bug in ObservableWrapper
             // where callComplete can fire before callEmit
             // remove this one the bug is fixed
-            TimerWrapper.setTimeout(() => { ObservableWrapper.callComplete(emitter); }, 0);
-          });
+            setTimeout(() => { emitter.complete(); }, 0);
+          }, (_) => {});
```

remove the rejection callback, it would swallow errors